com.unity.probuilder/Editor/EditorCore/DimensionsEditor.cs

```diff
@@ -46,10 +74,17 @@ void OnEnable()
 #else
             SceneView.onSceneGUIDelegate += OnSceneGUI;
 #endif
+            MeshSelection.objectSelectionChanged += OnObjectSelectionChanged;
+            ProBuilderMesh.elementSelectionChanged += OnElementSelectionChanged;
+            ProBuilderEditor.selectionUpdated += EditingMeshSelection;
```

Comment:
Shouldn't we keep the same convention as the two others callbacks? "On..." ?

Reply:
👍 
beef4a2